packages/vanilla/src/components/global-nav/top-nav/notifications/notifications.js

```diff
@@ -62,9 +61,7 @@ class Notifications extends Core {
   }
 
   addNotification(notification) {
-    if (notification instanceof Notification) {
-      this.list.addItem(notification);
-    }
+    this.list.addItem(notification);
```

Comment:
Notifications is not yet rewritten. It depends on several components. and so we needed to step into vanilla in order to allow it to render NotificationV1